infra/modules/service/main.tf

```diff
@@ -9,9 +9,9 @@ data "external" "whoami" {
   program = ["sh", "-c", "whoami | xargs -I {} echo '{\"value\": \"{}\"}'"]
 }
 
-data "external" "deploy_github_ref" {
-  program = ["sh", "-c", "git branch --show-current | xargs -I {} echo '{\"value\": \"{}\"}'"]
-}
+# data "external" "deploy_github_ref" {
```

Comment:
Maybe add a note with a ticket number or another comment of TODO, explaining how and when we'll fix this?

Reply:
That'll reset the review so I'll fast follow

Reply:
https://github.com/HHS/simpler-grants-gov/issues/3177

Reply:
Perfect, thanks!!